tests/jenkins/nrnivmodl-core.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+. /gpfs/bbp.cscs.ch/apps/hpc/jenkins/config/modules.sh
+module load intel/18.0.3 hpe-mpi/2.16
+CORENRN_TYPE=$1
+cd $WORKSPACE/testcorenrn
+rm -rf $CORENRN_TYPE
```

Comment:
`$CORENRN_TYPE` => could you use here full path? `$WORKSPACE/testcorenrn/$CORENRN_TYPE`